jax/api.py

```diff
@@ -112,6 +114,14 @@ def jit(fun: Callable, static_argnums: Union[int, Iterable[int]] = (),
       XLA's DeviceAssignment logic and is usually to use ``jax.devices()[0]``.
     backend: This is an experimental feature and the API is likely to change.
       Optional, a string representing the xla backend. 'cpu','gpu', or 'tpu'.
+    donate_argnums: Specify which arguments are "donated" to the computation.
+      When arguments are donated XLA will use their memory to store the result
```

Comment:
Maybe instead of "XLA will use ..." we can say "XLA can reuse their memory, e.g. to store the result of the computation."

However, I'm fine with merging first and making docstring tweaks in follow-ups, so this isn't blocking.

Reply:
Ah thanks, I hadn't updated the docstring to match the implementation. Have reworded.